### ISSUE TITLE
when you drag and drop an event to a date where the scrollbar must scroll for it glitches

### ISSUE THREAD

**Issue by EPKGames** [StephenChou1017/react-big-scheduler#97]
when you drag and drop an event to a date that is outside the dates that are showen within the view it does not drop on the right date.
so lets say i select quarter and I drag an event from 1/1/2019 to 1/3/2019 so that is in the same quarter but the scrollbar has to scroll for it when i drag it to that date and drop it, it is placed on a whole diffrent date.
this also happens on the example drag & drop scheduler on the example page from git hub.

**Comment by StephenChou1017** [comment]
@EPKGames 
Yeah, that is an issue, thanks for the feedback.

**Comment by Raven0uss** [comment]
@StephenChou1017 
Do you have some leads to solve this issue ?

During the DnD and Selection ; When we overflow the mouth to the `.event-container,` the parameter `element` of the function `getPos` called inside `doDrag` is `null`. I solved that, (cause it crashed in dev by the way) but it didn't fixed the issue. I don't know if you have some more informations to share to help me to solve this issue quickest as possible. 

Thanks for your job.

**Comment by StephenChou1017** [comment]
I have no idea temporarily, I tried to fix the issue(Moving an event causes scrollbar scrolls) first and then publish 0.2.6 but failed. The `endDrag` function in `DnDSource.js` is not called by react-DnD after scrollbar scrolls, I'll try it later.

**Comment by balpreetbrar** [comment]
@Raven0uss how are you handling this issue or still facing it?

**Comment by shivamjjha** [comment]
@Raven0uss There was a discussion on react-dnd repo on this
https://github.com/react-dnd/react-dnd/issues/38

Did you get around it somehow for this package?

**Comment by shivamjjha** [comment]
@StephenChou1017 @Raven0uss did you guys find any fix for this?

**Comment by hbatalhaStch** [comment]
@shivamjjha could you try [this maintained fork](https://github.com/hbatalhaStch/react-big-scheduler) and see if the problem persists

**Comment by shivamjjha** [comment]
@hbatalhaStch Yeah, I beleive so. I clone the repo locally and started `example` dir (http://localhost:8080/example/#/draganddrop)

Here is a recorded demo:


https://github.com/StephenChou1017/react-big-scheduler/assets/44032634/47f91e30-b00a-495c-9dd3-7ee0f1ee0a73

**Comment by hbatalhaStch** [comment]
> @hbatalhaStch Yeah, I beleive so. I clone the repo locally and started `example` dir (http://localhost:8080/example/#/draganddrop)
> 
> Here is a recorded demo:
> 
>  react-big-scheduler-examples-google-chrome-2023-07-29-21-12-49_WRe5PYW6.mp4

Sorry for the late reply, I haven't being getting much free time due to work,

Thanks for providing a recorded demo, it was extremely help as it prefectly shows the problem.
It's indeed a bug.
Could you please [open a new issue ](https://github.com/hbatalhaStch/react-big-scheduler/issues/new/choose) in my maintained fork, including the video, so I can track this bug there?

**Comment by shivamjjha** [comment]
Thanks for reaching out @hbatalhaStch 
I have opened a new issue https://github.com/hbatalhaStch/react-big-scheduler/issues/25

CC: @StephenChou1017 @Raven0uss @jennysong
